Code/Mantid/docs/source/interfaces/FilterEventUI.rst

```diff
@@ -0,0 +1,105 @@
+Filter Events Interface
+=======================
+
+.. contents:: Table of Contents
+  :local:
+  
+  
+Description
+-----------
+ 
+The Filter Events interface provides aid to users in order to set up event filter with different `Functionalities <http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html#functionalities>`_. 
```

Comment:
Is there a way I can reference SplittersWorkspace page as there seems to be a link for it http://www.mantidproject.org/SplittersWorkspace but I cannot find anything in docs directory